src/sentry/models/projectcodeowners.py

```diff
@@ -150,13 +150,16 @@ def merge_code_owners_list(self, code_owners_list):
 
         return merged_code_owners
 
-    def update_schema(self):
+    def update_schema(self, raw: str | None = None) -> None:
         """
         Updating the schema goes through the following steps:
         1. parsing the original codeowner file to get the associations
         2. convert the codeowner file to the ownership syntax
         3. convert the ownership syntax to the schema
         """
+        if raw:
+            self.raw = raw
```

Comment:
@NisanthanNanthakumar Why not always just set `self.raw = raw` even if it's `None`? Couldn't this lead to an attribute error unexpectedly?